base/src/com/google/idea/blaze/base/wizard2/ui/BlazeEditProjectViewControl.java

```diff
@@ -599,6 +600,9 @@ public void updateBuilder(BlazeNewProjectBuilder builder) {
               .add(
                   ScalarSection.builder(ImportSection.KEY)
                       .set(selectProjectViewOption.getSharedProjectView()))
+              .add(
```

Comment:
why is this needed? It results in having both import and try_import sections in the generated local projectview file.

Reply:
It is ok to use both `import` and `try_import`. It might be not ok to have some duplication or excessive defaults...

Could you please share more context for your inquiry?

Reply:
I do not know why it is needed to add a `try_import` in this function? If I understand correctly, this function creates a project view that imports the shared project view file selected in the import process. So `import` section should be enough and there is no need to `try_import` the same project view.

Reply:
@mai93 there might be "mandatory" and "optional" imports in any project view. One example of optional would be ```
try_import:
  ~/.some.local.projectview

```

(let's assume `~` is resolved to user home dir ;) )

Reply:
I see, I'm still not sure what the effect of having something like:

```
import hello/hello.bazelproject
try_import hello/hello.bazelproject
```

will this provide a different result from only
```
import hello/hello.bazelproject
```

Reply:
It is rather for
```
import hello/fail.if.not.present.bazelproject
try_import hello/ok.if.not.present.bazelproject
```

That can result in:
1. both imported
2. only `hello/fail.if.not.present.bazelproject` imported and accepting the fact that that `try_import hello/ok.if.not.present.bazelproject` is not there
3. failing if `hello/fail.if.not.present.bazelproject` is missing


Optional imports will allow engineers to have pre-defined custom import files both present or missing. 

Now it is possible for some of the users to benefit from custom contents while other users not affected even if they have no local customizations. For both groups of users the same `.bazelproject` can be stored in VCS now.

Reply:
so there is no additional value for having:
```
import hello/hello.bazelproject
try_import hello/hello.bazelproject
```
right? If so, then the change in this line can be reverted because it adds `try_import` section for the same project view file: `selectProjectViewOption.getSharedProjectView()` that already was added with `import` in line 600.

Reply:
Reverting here https://github.com/bazelbuild/intellij/pull/5958 @mai93